shell/platform/android/io/flutter/app/FlutterPlayStoreSplitApplication.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.app;
+
+import com.google.android.play.core.splitcompat.SplitCompatApplication;
+import androidx.annotation.CallSuper;
+import io.flutter.embedding.engine.dynamicfeatures.PlayStoreDynamicFeatureManager;
+import io.flutter.FlutterInjector;
+
+/**
+ * Flutter's extension of {@link SplitCompatApplication} that injects a
+ * {@link PlayStoreDynamicFeatureManager} with {@link FlutterInjector} to enable Split AOT
+ * Flutter apps.
+ *
+ * To use this class, either have your custom application class extend FlutterPlayStoreSplitApplication
+ * or use it directly in the app's AndroidManifest.xml by adding the following line:
+ * <pre>
+ * {@code
```

Comment:
Does double wrapping <pre> and `{@code}` work?

Reply:
I believe the pre indicates that the formatting should not change, while the @code indicates that the contents is code and should be highlighted as such